src/inspector_agent.cc

```diff
@@ -658,9 +686,21 @@ bool Agent::StartIoThread(bool wait_for_connect) {
   }
 
   v8::Isolate* isolate = parent_env_->isolate();
+  HandleScope handle_scope(isolate);
+
+  // Enable tracking of async stack traces
+  if (!enable_async_hook_function_.IsEmpty()) {
+    Local<Function> enable_fn = enable_async_hook_function_.Get(isolate);
```

Comment:
Oh you mean tests. Yeah i'm alright with that. We are exploring ways to make inspector tests easier for both the current WS and local JS version in #14460, so we can always add it later!